tests/rust/test_repo.py

```diff
@@ -12,7 +31,40 @@ def test_default_repo_render(tmp_path):
         output_dir=tmp_path,
     )
 
-    actual = DiffDir(tmp_path)
-    expected = DiffDir(TEMPLATE_ROOT / "tests/rust/repo/expecteds/default")
+    actual = DiffPath(tmp_path / "repository")
+    expected = DiffPath(DEFAULT_REPO_DIR)
+
+    assert actual == expected
+
+
+def test_agpl(tmp_path):
+    config = {"default_context": {"license": "AGPL-3.0-only"}}
+    config_file = tmp_path / "config.yaml"
+    with config_file.open(mode="w") as f:
+        yaml.dump(config, f)
+
+    cookiecutter(
+        str(TEMPLATE_ROOT),
+        directory="rust/repo",
+        no_input=True,
+        output_dir=tmp_path,
+        config_file=config_file,
+    )
+
+    actual_path = tmp_path / "repository"
+    actual = DiffPath(actual_path)
+    actual.exclude(("Cargo.toml", "deny.toml", "LICENSE"))
+    expected = DiffPath(DEFAULT_REPO_DIR)
 
     assert actual == expected
+
+    cargo_toml = toml.load(actual_path / "Cargo.toml")
+    assert cargo_toml["workspace"]["package"]["license"] == "AGPL-3.0-only"
+
+    deny_toml = toml.load(actual_path / "deny.toml")
+    assert deny_toml["licenses"]["allow"] == AGPL_ALLOW_LICENSES
+    assert deny_toml["licenses"]["copyleft"] == "allow"
+    assert deny_toml["licenses"]["allow-osi-fsf-free"] == "both"
```

Comment:
The downside with this approach is that it doesn't check the formatting of the toml and I know during development that was one of the review comments.
It does ensure the toml is parsable.